### PR TITLE
feat: Add on-hover tooltips for tx details

### LINKED ISSUE #3118: Add on-hover tooltips for tx details
Add on-hover tooltips to educate users about the line items in transaction details

We already have a component for this called `MouseoverTooltip`. Here is an [example](https://github.com/Uniswap/interface/blob/9e1a775c137ac826ba9affe0f2d702ed61f386c9/src/components/CurrencyInputPanel/FiatValue.tsx#L46) of how to use it.

We should add a tooltip to the text area of each of the four line items below.

Screen Shot 2022-01-13 at 10 16 34 PM

**Text strings to use in the tooltip**
- [ ]  Expected Output
    - “The amount you expect to receive at the current market price. You may receive less or more if the market price changes while your transaction is pending.”
- [ ]  Price Impact
    - “The impact your trade has on the market price of this pool.”
- [ ]  Minimum received after slippage (0.50%)
    - “The minimum amount you are guaranteed to receive. If the price slips any further, your transaction will revert.”
- [ ]  Network fee
    - “The fee paid to miners who process your transaction. This must be paid in {ETH | MATIC}.”   (switch native token depending on network)

### PR DESCRIPTION
**Pull request by 0xlucius**
Closes #3118

### CODE REVIEW

**Comment by willhennessy** [comment]
Thanks lucius! This looks good to me, but let me ask an engineer to give a quick review. @JFrankfurt can you ptal?

**Comment by JFrankfurt** [comment]
@0xlucius can you get this working w/ <Trans> instead of t``?

**Comment by 0xlucius** [comment]
@JFrankfurt Changed t macros to <Trans> and removed the { t } import.

**Comment by JFrankfurt** [comment]
@callil, @maggievalentine, or @ppphhhiiilll - implementation here lgtm, but what do you guys think on design? Having larger text in a tooltip than the text being hovered feels off to me.

image

**Comment by callil** [comment]
Looks pretty good to me.

**Comment by willhennessy** [comment]
@JFrankfurt can we approve and merge? cal gave the lgtm above!

**Comment by willhennessy** [comment]
one nit:  the on-hover tip triggers if you try to hover over the info bubble. It should only appear on the expanded labels.

Steps to repro:
1. enter a price and get a quote
2. hover over the info 'i' to see trade details
3. quickly move your mouse to hover over expected output

observed:  the tooltip for expected output appears
intended:  the tooltip for expected output should NOT appear here because it would be a hover on a hover

Screen Shot 2022-04-14 at 6 12 08 PM

File: src/components/swap/AdvancedSwapDetails.tsx [hunk 1]
```diff
@@ -106,9 +139,19 @@ export function AdvancedSwapDetails({ trade, allowedSlippage, syncing = false }:
...
         {!trade?.gasUseEstimateUSD || !chainId || !SUPPORTED_GAS_ESTIMATE_CHAIN_IDS.includes(chainId) ? null : (
           <RowBetween>
-            <ThemedText.SubHeader color={theme.text3}>
-              <Trans>Network Fee</Trans>
-            </ThemedText.SubHeader>
+            <MouseoverTooltip
+              text={
+                <Trans>
+                  The fee paid to miners who process your transaction. This must be paid in{' '}
+                  {CHAIN_INFO[chainId ?? 1].nativeCurrency.symbol}.
```
Comment (zzmp) [inline src/components/swap/AdvancedSwapDetails.tsx:146]: Instead of using `CHAIN_INFO`, use `useNativeCurrency()`.symbol.

**Review by zzmp — APPROVED** [review]

File: src/components/swap/AdvancedSwapDetails.tsx (hunk 1 above)
Comment (leggechr) [reply src/components/swap/AdvancedSwapDetails.tsx]: Done.

**Review by leggechr — APPROVED** [review]

**Pull request merged**